### PR TITLE
Remove System.Security.Cryptography.Cng Dependency

### PR DESCRIPTION
**Pull request by TanayParikh**

### CODE REVIEW

**Review by Tratcher — APPROVED** [review]

**Comment by Tratcher** [comment]
How did you find this?

**Comment by TanayParikh** [comment]
> How did you find this?

`System.Security.Cryptography.Cng` was using a stale dotnet/runtime dependency, @BrennanConroy / @dougbu suggested we may be able to remove the package outright now.

**Review by Pilchie — APPROVED** [review]

**Review by wtgodbe — APPROVED** [review]

File: eng/PackageOverrides.txt [hunk 1]
```diff
@@ -2,7 +2,6 @@ Microsoft.Win32.Registry|5.0.0
 System.Diagnostics.EventLog|5.0.0
 System.IO.Pipelines|5.0.0
 System.Security.AccessControl|5.0.0
-System.Security.Cryptography.Cng|5.0.0
```
Comment (dougbu) [inline eng/PackageOverrides.txt:5]: Revert changes in this file. This file is about our 5.0.0 release.

**Review by dougbu — APPROVED** [review]

File: eng/PackageOverrides.txt [hunk 2]
```diff
@@ -2,7 +2,6 @@ Microsoft.Win32.Registry|5.0.0
 System.Diagnostics.EventLog|5.0.0
 System.IO.Pipelines|5.0.0
 System.Security.AccessControl|5.0.0
```
Comment (TanayParikh) [inline eng/PackageOverrides.txt:4]:
```suggestion
System.Security.AccessControl|5.0.0
System.Security.Cryptography.Cng|5.0.0
```

**Comment by TanayParikh** [comment]
@Tratcher 

https://github.com/dotnet/aspnetcore/blob/73d9fa93e109474982db9fdd52d309c9313296de/src/Servers/HttpSys/test/FunctionalTests/ServerTests.cs#L386-L415

is now failing with:
```
Assert.Throws() Failure
Expected: typeof(System.Net.Http.HttpRequestException)
Actual:   (No exception was thrown)
```

Any reason to think this may be related to the changes, or would it be safe to quarantine that test?

https://dev.azure.com/dnceng/public/_build/results?buildId=1126028&view=logs&jobId=3f6d4e0f-1b71-56b5-361e-d95b6e6da15a&j=3f6d4e0f-1b71-56b5-361e-d95b6e6da15a&t=d5bf30bc-9e5a-596c-d1c8-39e2dfa42d49

**Comment by Tratcher** [comment]
No relation, go ahead and quarantine it.

**Pull request merged**